docs/UserGuide.md

```diff
@@ -167,13 +167,35 @@ Example |  Outcome
 ### 3.4. Chore
 
 #### 3.4.1. Adding a chore: `addchore`
+Adds a chore to the chore list in Kitchen Helper.
+
+__Format:__ `addchore <task_description> /by <deadline>`  
+
+__Example of usage:__  
+* `addchore buy groceries /by Monday 12pm`  
+Creates a new chore called `buy groceries` which contains the deadline `Monday 12pm`.
+
+
 
 #### 3.4.2. List chore: `listchore`
+Displays all the items currently in the chore list in Kitchen Helper.
+__Format:__ `listchore`  
+
+__Example of usage:__  
+* `listchore`  
+
 
 #### 3.4.3. Delete a chore: `deletechore`
+Deletes the chore specified by the index in the chore list in Kitchen Helper. The index of the chore can be found by displaying the list of chores.
+__Format:__ `deletechore <index_to_delete>`  
 
-#### 3.4.3. Search for chore: `searchchore`
-Search for chores based on a given keyword.  
+__Example of usage:__  
```

Comment:
Good job in adding the example of usages for the different commands! Perhaps, the output of the command could be provided to let the users see what kind of output they might receive? :)